src/containers/AccountConnection.jsx

```diff
@@ -260,6 +267,10 @@ class AccountConnection extends Component {
 
             { !success && <KonnectorSync
               date={account && account.lastSync}
+              error={error}
+              editing={editing}
```

Comment:
I was not understanding why `editing` was passed as property here. But it appears that the above condition miss a parameter, see my PR https://github.com/cozy/cozy-collect/pull/205. Once merged, we will not need to pass `editing` anymore here.